test/zdtm/static/cpu-affinity0.c

```diff
@@ -0,0 +1,42 @@
+#include <errno.h>
+#include <stdlib.h>
+#include <sched.h>
+
+#include "zdtmtst.h"
+
+const char *test_doc   = "Check that with-cpu-affinity option can restore cpu affinity";
+const char *test_author    = "Zhou jie <zhoujie133@huawei.com>, He jingxian <hejingxian@huawei.com>";
+
+int main(int argc, char **argv)
+{
+   cpu_set_t old;
+   cpu_set_t new;
+
+   test_init(argc, argv);
+
+   CPU_ZERO(&old);
+   CPU_ZERO(&new);
+   
+   /* test only 0 core because of CI test env limited */
+   CPU_SET(0, &old);
+
+   if (sched_setaffinity(getpid(), sizeof(old), &old) < 0) {
+       pr_perror("Can't set old cpu affinity! errno: %d", errno);
```

Comment:
`pr_perror` already prints the value of `errno`:
https://github.com/checkpoint-restore/criu/blob/b3a950bf7c7c286e781eb02230af21324853b4fb/criu/include/log.h#L61